src/mds/CDir.cc

```diff
@@ -2005,7 +2005,8 @@ void CDir::_omap_fetched(bufferlist& hdrbl, map<string, bufferlist>& omap,
 {
   LogChannelRef clog = mdcache->mds->clog;
   dout(10) << "_fetched header " << hdrbl.length() << " bytes "
-	   << omap.size() << " keys for " << *this << dendl;
+	   << omap.size() << " keys for " << *this
+           << " " << complete << dendl;
```

Comment:
I think it would be better, instead of just logging the boolean "complete" value, log a "(complete)" or "(incomplete)" string.